src/components/Settings/Settings.tsx

```diff
@@ -75,7 +93,7 @@ const Settings = () => {
           </div>
         </div>
         <div className="text-sm flex justify-center">
-          Note: This is just the UI, and is not currently connected to backend
+          Note: App will be reloaded on save
```

Comment:
Mind explaining the thought behind this - 
- What do you mean by reloading?  
- Do we reload if we change the name info of our node, or any node? 
- Will we display the splash page whenever we update a node?

Reply:
Maybe this is something I missed in a previous PR, but how does the app reload? And for the reload to be useful, we have to be able to guarantee that it takes longer than the device restart. I think the web client handles this with a "reconnect" command

Reply:
Known bug with #166, no changes on this PR